formaldoc/_root/post-form.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: |-
+  Printit
+
+url: |-
+  /
+
+services:
+- method: |-
+    POST_FORM
```

Comment:
Not sure to understand why it's a POST_FORM instead of POST

Reply:
Backward compatible way of getting attachments with no modern javascript.